service/microservice.py

```diff
@@ -22,11 +19,21 @@ def start_service():
 def cloud_storage_service(_req, resp):
     """Send the request through to the cloudstorage microservice"""
     path = urlparse(_req.uri).path[1:]
+
+    # determine whether to use amazon s3 or azure blob storage
+    # Use 'az' at beginning of the path to signal that file is from azure
+    # eg. https://domain.com/az/some_file.pdf
+    microservice_url = os.environ.get('CLOUDSTORAGE_URL') # default to s3
+
+    if path[0:3] == 'az/':
+        microservice_url = microservice_url.replace('1.0', '2.0', 1)
```

Comment:
For clarity, would it be possible to use endpoints with names versus numbers?

Reply:
It's definitely possible, but might be a little a late in the game to change.  We would have to modify the urls for other projects.  The reason I went with versioning is so that we aren't tied to the service provider and so that we don't tip our hand as to what storage provider we are using.  I'm guessing that we'll drop  bucketeer at some point when everybody has transitioned over to azure.

Reply:
Yes, I see versioning used in the storage microservice, it make sense.